src/avs/AVSDetails.jsx

```diff
@@ -1,330 +1,164 @@
-import {
-  Button,
-  Card,
-  CardBody,
-  CardHeader,
-  Link,
-  Tab,
-  Tabs
-} from '@nextui-org/react';
-import { useCallback, useEffect, useMemo } from 'react';
-import { useLocation } from 'react-router-dom';
 import { useMutativeReducer } from 'use-mutative';
-import { useServices } from '../@services/ServiceContext';
-import GraphTimelineSelector from '../shared/GraphTimelineSelector';
 import { reduceState } from '../shared/helpers';
-import { assetFormatter, formatNumber } from '../utils';
-import AVSTotalValueOvertime from './AVSTotalValueOvertime';
-import LSTDistribution from './LSTDistribution';
-import Operators from './Operators';
-import OperatorsOvertime from './OperatorsOvertime';
-import RestakersOverTime from './RestakersOverTime';
-import RestakingLeaderboard from './RestakingLeaderboard';
-import { strategiesData } from './strategies.mapping';
+import { useEffect } from 'react';
+import { Skeleton, Tab, Tabs } from '@nextui-org/react';
+import { useServices } from '../@services/ServiceContext';
+import { useParams } from 'react-router-dom';
+import AVSDetailsHeader from './AVSDetailsHeader';
+import AVSDetailsTVLTab from './AVSDetailsTVLTab';
+import AVSDetailsOperatorsTab from './AVSDetailsOperatorsTab';
+import { BEACON_STRATEGY, EIGEN_STRATEGY } from './helpers';
+import { useTailwindBreakpoint } from '../shared/useTailwindBreakpoint';
+import { formatETH, formatNumber } from '../shared/formatters';
```

Comment:
Fix import order to silence the linter:

```suggestion
import { BEACON_STRATEGY, EIGEN_STRATEGY } from './helpers';
import { formatETH, formatNumber } from '../shared/formatters';
import { Skeleton, Tab, Tabs } from '@nextui-org/react';
import AVSDetailsHeader from './AVSDetailsHeader';
import AVSDetailsOperatorsTab from './AVSDetailsOperatorsTab';
import AVSDetailsTVLTab from './AVSDetailsTVLTab';
import { reduceState } from '../shared/helpers';
import { useEffect } from 'react';
import { useMutativeReducer } from 'use-mutative';
import { useParams } from 'react-router-dom';
import { useServices } from '../@services/ServiceContext';
import { useTailwindBreakpoint } from '../shared/useTailwindBreakpoint';
```